packages/cache/package.json

```diff
@@ -10,7 +10,7 @@
     "test": "ava",
     "test:c8": "c8 $C8_OPTIONS ava --config=ava-nesm.config.js",
     "test:xs": "exit 0",
-    "lint-fix": "yarn lint:eslint --fix",
+    "lint-fix": "yarn lint:eslint --report-unused-disable-directives --fix",
```

Comment:
I trust we are here because we can’t consolidate this flag in config. Is there some other reason?

Reply:
Thanks for asking. I believe ESlint lacked a flag when that commit was made but now there is a config option and we've since enabled it:
https://github.com/Agoric/agoric-sdk/blob/e134cdc651b8bb6af28a9a8a15f25b5cb37c5edc/.eslintrc.cjs#L81

I'll revert all the `lint-fix` line changes